### PR TITLE
More actions dropdown button toggling on bulk actions

### PR DESCRIPTION
**Pull request by Tijani-Dia**
Targets #8045.

The goal of this PR is to hide the 'more' dropdown button in listing pages when a child page is selected to perform bulk actions.

* Do the tests still pass? Yes
* Does the code comply with the style guide? 
* For front-end changes: Did you test on all of Wagtail’s supported environments? No
    * **Please list the exact browser and operating system versions you tested**. Firefox, Chrome
    * **Please list which assistive technologies you tested**. None

### CODE REVIEW

**Comment by zerolab** [comment]
@Tijani-Dia could you have a look at the lint failures?

**Comment by Tijani-Dia** [comment]
@zerolab which lint failures?

**Comment by zerolab** [comment]
@Tijani-Dia ah, sorry. Looks like the Circle CI tests (which usually are linting + ui_tests) failed on `ImportError: cannot import name 'soft_unicode' from 'markupsafe'`. Which should've been fixed by #8014 🤔

Not one for you to worry about (sorry again)

File: client/src/entrypoints/admin/bulk-actions.js [hunk 1]
```diff
@@ -16,6 +17,30 @@ const checkedState = {
   shouldShowAllInListingText: true,
 };
 
+/**
+ * Hides the 'more' dropdown button in listing pages.
+ */
+function hideMoreActionsDropdownBtn() {
+  const MoreActionsDropdown = document.querySelector(
```
Comment (zerolab) [inline client/src/entrypoints/admin/bulk-actions.js:24]: nitpick: `MoreActionsDropdown` -> `moreActionsDropdown`

```suggestion
  const moreActionsDropdown = document.querySelector(
```

my preference would have been to have a single `toggleMoreActionsDrodown` function that accepts a show/hide parameter, but this is fine too

File: client/scss/components/_dropdown.scss [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // =============================================================================
 // Listing view smaller dropdowns
 // =============================================================================
+.c-dropdown.hidden {
```
Comment (zerolab) [inline client/scss/components/_dropdown.scss:4]: Adding `.hidden` to the `.c-dropdown` element does the job. do we need the extra specificity?

**Review by zerolab — CHANGES_REQUESTED** [review]
Hey @Tijani-Dia,

thanks for this. Left a couple of minor comments.
Otherwise, it does the job nicely

File: client/scss/components/_dropdown.scss (hunk 2 above)
Comment (Tijani-Dia) [reply client/scss/components/_dropdown.scss:4]: Do you mean that we don't need this specific rule?

File: client/src/entrypoints/admin/bulk-actions.js (hunk 1 above)
Comment (Tijani-Dia) [reply client/src/entrypoints/admin/bulk-actions.js]: I've replaced the previous functions by a unique `toggleMoreActionsDropdownBtn` function.

**Comment by Tijani-Dia** [comment]
Thanks @zerolab!

I'm having a lint failure but I don't really know what the linter wants me to do... Do you know what it expects?

File: client/scss/components/_dropdown.scss (hunk 2 above)
Comment (zerolab) [reply client/scss/components/_dropdown.scss:4]: Indeed. simply adding the `hidden` class to the html element already hides it

File: client/src/entrypoints/admin/bulk-actions.js [hunk 3]
```diff
@@ -16,6 +17,21 @@ const checkedState = {
...
+/**
+ * Toggles the 'more' dropdown button in listing pages.
+ * @param {boolean} show - Determines if the button should be shown or not.
+ */
+function toggleMoreActionsDropdownBtn(show) {
+  const moreActionsDropdown = document.querySelector(
+    MORE_ACTIONS_DROPDOWN_BUTTON_SELECTOR,
+  );
+  if (moreActionsDropdown !== null) {
+    show === true
```
Comment (zerolab) [inline client/src/entrypoints/admin/bulk-actions.js:29]: this is where the linter is complaining. change it to an `if (show === true) { ... }`?

File: client/scss/components/_dropdown.scss (hunk 2 above)
Comment (Tijani-Dia) [reply client/scss/components/_dropdown.scss:4]: > simply adding the hidden class to the html element already hides it

I've tried it but it doesn't seem to work.

File: client/src/entrypoints/admin/bulk-actions.js (hunk 3 above)
Comment (Tijani-Dia) [reply client/src/entrypoints/admin/bulk-actions.js]: Alright, I've changed it.

File: client/scss/components/_dropdown.scss (hunk 2 above)
Comment (zerolab) [reply client/scss/components/_dropdown.scss:4]: ok. must've been a fluke on my side. 🤔 let's keep it in. thank you for revisiting

**Review by zerolab — APPROVED** [review]
LGTM!

**Pull request merged**